scripts/update_rpc.py

```diff
@@ -0,0 +1,150 @@
+""" Script to generate a python const file with public chain RPC endpoints from the Ethereum
+Foundation ethereum-lists/chains repo.
+
+Usage
+-----
+python scripts/update_rpc.py
+"""
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+from urllib.parse import urljoin
+
+import requests
+from pydantic import BaseModel
+
+from ape.logging import logger
+
+CHAIN_IDS = {
+    "base": {
```

Comment:
do they have any of these:

1. avalanche
2. arbitrum
3. bsc
4. fantom

im just looking through some of the other plugins we have

Reply:
[29f449b](https://github.com/ApeWorX/ape/pull/1866/commits/29f449be15c76966192b177f21b2014b02319b6d)